plugins/templates/listTags.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const boom = require('boom');
+const joi = require('joi');
+const schema = require('screwdriver-data-schema');
+const listSchema = joi.array().items(schema.models.templateTag.base).label('List of templates');
+const nameSchema = joi.reach(schema.models.templateTag.base, 'name');
+
+module.exports = () => ({
+    method: 'GET',
+    path: '/templates/{name}/tags',
+    config: {
+        description: 'Get all template tags for a given template name with pagination',
```

Comment:
We actually don't have pagination right now: https://github.com/screwdriver-cd/screwdriver/issues/233

Reply:
@ScottyFillups  We should remove the `with pagination`